guides/release/tutorial/20.md

```diff
@@ -0,0 +1,183 @@
+So far, we've been hard-coding everything into our `<Rental>` component. But that's probably not very sustainable, since eventually, we want our data to come from a server instead of hard-coded values. Let's go ahead and move some of those values out of the component in preparation for that.
```

Comment:
I did an editing pass here: https://github.com/emberjs/rfcs/blob/0e3c134048396202462c09aea336a319bf4e9bed/text/0523-model-argument-for-route-templates.md#how-we-teach-this